docs/floating-current.md

```
@@ -0,0 +1,110 @@
+// Legacy (ui-kit/Tooltip)
+function calculatePlacement(target: DOMRect, tooltip: DOMRect) {
+  // Manual boundary checks
+}
+
+// Modern (floating/Tooltip)
+const { x, y, refs } = useFloating({
+  middleware: [autoPlacement(), offset(4)],
+});
+```
+
+2. **Context System Fragmentation**
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2; Style: 1/1/1
Ordered list item prefix

(MD029, ol-prefix)